doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -855,7 +855,10 @@ A subscription is represented by a JSON object with the following fields:
 + `id`: Subscription unique identifier. Automatically created at creation time.
 + `subject`: It is an object that describes the subject of the subscription.
 + `notification`: It is an object that describes the notification received by the subscriber.
-+ `duration`: Duration of the subscription in ISO8601 format. Infinite if not specified.
++ `expires`: Subscription expiration date in ISO8601 format. Permanent subscriptions must omit
+   this field.
```

Comment:
As far as I understand, the exact encoding is part of the ISO8601 string, so it can be univocally identified (or generated) by the client. I mean, if you use:

2016-04-05T14:00:00.00Z

you are meaning a date in UTC format while if you use

1994-11-05T08:15:30-05:00

you are referring to a time in GMT-5 timezone.

So, no ambiguety is possible within the ISO8601 standard.

Moreover, currently Orion generates dates in UTC but 1) this may change in the future, 2) we haven't defined yet what to do with dates provided by clients not using UTC (and maybe it is easy to process them if the library we use from parsing convert them to the right timestamp time_t for free). Moreover, both can be considered subtle detailt related with our particular NGSIv2 implementation.

Bottom line: I think is ok have the current wording in the spec. However, as I understand the whole specification will be reviewed once NGSIv2 get fully implemented, we have an opportunity in the future to change it if it doesn't match the final implementation in Orion.

Regarding emphasizing "date", I also think that it is pretty clear with the current working. Adding more stuff could make understanding harder (*excusatio non petita accusatio manifesta* ;).